front/app/components/CampaignConsentForm/feedback.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import Error from 'components/UI/Error';
+import {
+  Text,
+  Title,
+  Box,
+  colors,
+  Icon,
+} from '@citizenlab/cl2-component-library';
+import { scrollToElement } from 'utils/scroll';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+type FeedbackProps = {
+  successMessage: string;
+  showFeedback: false | 'success' | 'error';
+  errorMessage: string;
+  closeFeedback: () => void;
+};
+
+const Feedback = ({
+  successMessage,
+  errorMessage,
+  showFeedback,
+  closeFeedback,
+}: FeedbackProps) => {
+  const [feedbackIsVisible, setFeedbackIsVisible] = useState(false);
+
+  useEffect(() => {
+    if (feedbackIsVisible) {
+      scrollToElement({ id: 'feedback' });
+    }
+  }, [feedbackIsVisible]);
+
+  useEffect(() => {
```

Comment:
Would this work instead of this `useEffect`?

```ts
const [feedbackIsVisible, setFeedbackIsVisible] = useState(!!showFeedback);
```

Reply:
that would not update the state when the prop change on the parent component.
so after you save, the show feedback prop is reseted.

Reply:
thanks that looks much better